### ISSUE TITLE
Minor tokenization issue with lowercase 'i' + contraction

### ISSUE THREAD

**Issue by NSchrading** [honnibal/spaCy#26]
spaCy correctly tokenizes capital "I" + contraction ('d, 'm, 'll, 've) e.g.:
```python
from spacy.en import English
nlp = English()
tok = nlp("I'm")
print([x.lower_ for x in tok])
```

``` >>> ['i', "'m"]```

but when the "I" is a lowercase ("i") it does not tokenize into two tokens:
```python
from spacy.en import English
nlp = English()
tok = nlp("i'm")
print([x.lower_ for x in tok])
```

``` >>> ["i'm"]```

Not a big deal, and this may be the intent, since we don't know if the user meant capital "I", but I can't think of any problems that would happen if it tokenized the lowercase version into two.

**Comment by honnibal** [comment]
Thanks, this is a gap in the tokenization special-case data. I'll fix this. It should also handle stuff like "im".

**Comment by honnibal** [comment]
Fixed in version 0.70.
